src/components/items/Items.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Item from './Item';
+
+const Items = ({ items }) => {
```

Comment:
this is mostly just a code-structure question-- Items gets items and loops through them to create an array of Item elements-- this feels a bit too vague, like what is an Item? It feels like I have no idea what your code necessarily is about, and being just a bit more specific would be helpful.